src/Interpolations.jl

```diff
@@ -28,11 +28,12 @@ using WoodburyMatrices, Ratios, AxisAlgorithms
 import Base: convert, size, getindex, gradient, promote_rule
 
 abstract InterpolationType
+immutable NoInterp <: InterpolationType end
 abstract GridType
 immutable OnGrid <: GridType end
 immutable OnCell <: GridType end
 
-typealias DimSpec{T} Union(T,Tuple{Vararg{T}})
+typealias DimSpec{T} Union(T,Tuple{Vararg{Union(T,NoInterp)}},NoInterp)
```

Comment:
I didn't know that was a thing, but when I re-read the manual now it seems parentheses aren't used anywhere anymore :P

I guess this is because we now prefer using the parametric type directly rather than through a constructor? (Will the constructor be deprecated soon-ish?)